### PR TITLE
Send charge tags as BSD source value

### PR DESCRIPTION
**Pull request by anero**

### CODE REVIEW

File: spec/notifiers/blue_state_notifier_spec.rb [hunk 1]
```diff
@@ -27,4 +27,22 @@
...
+    (0...5).each do
+      charge.tags << (tag = create(:tag, organization: organization))
+      tag_names << "takecharge:#{tag.name}"
+    end
+    charge.save!
+
+    expect(BlueStateDigital::Contribution).to receive(:new) do |options|
+      sources = options[:source]
+      expect(sources.length).to eq tag_names.length
+      tag_names.each { |tag_name| expect(sources).to include(tag_name) }
```
Comment (lavaturtle) [inline spec/notifiers/blue_state_notifier_spec.rb]: you might be able to write this as <code>expect(sources).to contain_exactly(tag_names)</code>

**Comment by lavaturtle** [comment]
:+1:

File: spec/notifiers/blue_state_notifier_spec.rb (hunk 1 above)
Comment (anero) [inline spec/notifiers/blue_state_notifier_spec.rb]: Sadly that matcher is not available on Rspec 2.14 (which is the version prague is using)

Comment (woodhull) [inline spec/notifiers/blue_state_notifier_spec.rb]: update rspec?

**Pull request merged**